modules/temp-channels/config.json

```
@@ -308,6 +308,55 @@
         "en": "The message to be sent, if a user edited their channel, but it failed"
       },
       "type": "string"
+    },
+    {
+      "name": "settingsChannel",
+      "humanName": {
+        "de": "Einstellungskanal",
+        "en": "Settings channel"
+      },
+      "default": {
+        "en": ""
+      },
+      "description": {
+        "en": "You can set a channel here in which the settings menu should be created. Leave this field empty, if you don't want to use this feature.",
+        "de": "Gebe hier die ID des Kanals an, in welcher das Einstellungsmenü erstellt werden soll. Lass dieses Feld leer, wenn du diese Funktion nicht verwenden willst."
+      },
+      "type": "channelID",
+      "content": "GUILD_TEXT",
```

This needs to be an array
```suggestion
      "content": ["GUILD_TEXT"],
```

---

Same in the "category" field